tests/wpt-jsdom/to-run.yaml

```diff
@@ -11,6 +11,14 @@ name_test_case_552-manual.html:
   [fail, getComputedStyle pseudo selector not implemented]
 name_test_case_553-manual.html:
   [fail, getComputedStyle pseudo selector not implemented]
+name_test_case_596-manual.html:
```

Comment:
Do we have tracking issues for those on WPT side? 

Let's update the WPT repo in here as well to ensure these bugs are in their latest version.

Reply:
TODO: reference https://github.com/web-platform-tests/wpt/issues/40280